app/requests/createGmProductsConstraints.js

```diff
@@ -88,58 +88,51 @@ const constraints = {
   name: {
     presence: {
       allowEmpty: false,
-      message: "^Please enter a Product Name",
+      message: "^Enter a Product Name",
```

Comment:
`Enter Product Name` and not `Enter a Product Name`.

@Subbu-betalectic please match these names with what you have written in the mars issue.
https://mars.betalectic.com/apps/product-wiredup/wiredup/-/issues/388

Reply:
Ok @ShubhamJoshi-1994